src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -56,210 +155,194 @@ public new IEnumerable ItemsSource
                     return;
                 }
 
-                if (_itemsLayouter == null) return;
+                if (itemsLayouter == null) return;
 
                 if (InternalItemSource != null) InternalItemSource.Dispose();
-                InternalItemSource = ItemsSourceFactory.Create(this, _itemsLayouter);
-                NeedInitalizeLayouter = true;
+                InternalItemSource = ItemsSourceFactory.Create(this, itemsLayouter);
```

Comment:
ItemsSourceFactory.Create's first parameter is IEnumerable itemsSource.
Is it correct if "this"(CollectionView) is sent to ItemsSourceFactory.Create's first argument?

Reply:
This `Create()` will call 
```cs
IGroupableItemSource Create(CollectionView colView, ItemsLayouter layouter)
```

Reply:
Thank you, I missed it T.T